### PR TITLE
chore: Fix linter findings for `revive:unused-receiver` in `plugins/inputs/[a-e]`

### PR DESCRIPTION
**Pull request by zak-pawel**
## Summary

Address findings for [revive:unused-receiver](https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md#unused-receiver) in `plugins/inputs/[a-e]*`.

It is only part of the bigger work for https://github.com/influxdata/telegraf/issues/16190.
After all findings of this type in whole project are handled, we can enable `revive:unused-receiver` rule in `golangci-lint`.

## Checklist

- [x] No AI generated code was used in this PR

### CODE REVIEW

File: plugins/inputs/elasticsearch/elasticsearch.go [hunk 1]
```diff
@@ -159,7 +159,7 @@ func (e *Elasticsearch) Init() error {
 	return nil
 }
 
-func (e *Elasticsearch) Start(_ telegraf.Accumulator) error {
+func (*Elasticsearch) Start(_ telegraf.Accumulator) error {
```
Comment (srebhan) [inline plugins/inputs/elasticsearch/elasticsearch.go:162]:
```suggestion
func (*Elasticsearch) Start(telegraf.Accumulator) error {
```

File: plugins/inputs/elasticsearch_query/elasticsearch_query.go [hunk 2]
```diff
@@ -89,7 +89,7 @@ func (e *ElasticsearchQuery) Init() error {
 	return nil
 }
 
-func (e *ElasticsearchQuery) Start(_ telegraf.Accumulator) error {
+func (*ElasticsearchQuery) Start(_ telegraf.Accumulator) error {
```
Comment (srebhan) [inline plugins/inputs/elasticsearch_query/elasticsearch_query.go:92]:
```suggestion
func (*ElasticsearchQuery) Start(telegraf.Accumulator) error {
```

File: plugins/inputs/execd/shim/shim_test.go [hunk 3]
```diff
@@ -105,11 +101,11 @@ func (i *testInput) Gather(acc telegraf.Accumulator) error {
 	return nil
 }
 
-func (i *testInput) Start(_ telegraf.Accumulator) error {
+func (*testInput) Start(_ telegraf.Accumulator) error {
```
Comment (srebhan) [inline plugins/inputs/execd/shim/shim_test.go:104]:
```suggestion
func (*testInput) Start(telegraf.Accumulator) error {
```

File: plugins/inputs/execd/shim/shim_test.go [hunk 4]
```diff
@@ -157,11 +149,11 @@ func (i *serviceInput) Gather(acc telegraf.Accumulator) error {
 	return nil
 }
 
-func (i *serviceInput) Start(_ telegraf.Accumulator) error {
+func (*serviceInput) Start(_ telegraf.Accumulator) error {
```
Comment (srebhan) [inline plugins/inputs/execd/shim/shim_test.go:152]:
```suggestion
func (*serviceInput) Start(telegraf.Accumulator) error {
```

**Review by srebhan — COMMENTED** [review]
Thanks @zak-pawel! Looks good, only a few suggestions to get rid of some more underscores.

**Review by srebhan — APPROVED** [review]
Thanks @zak-pawel!

**Review by DStrand1 — APPROVED** [review]

**Pull request merged**